packages/flutter_tools/lib/src/base/extension_host.dart

```diff
@@ -0,0 +1,128 @@
+import 'dart:async';
+import 'dart:isolate';
+
+import 'package:flutter_tool_api/extension.dart';
+import 'package:yaml/yaml.dart';
+
+import '../convert.dart';
+import '../globals.dart';
+import 'file_system.dart';
+import 'platform.dart';
+
+/// Generates and caches a snapshot of the tool extension apis to apply.
+class CrossIsolateShim {
+  CrossIsolateShim() {
+    // HARDCODED.
+    const String path = '/Users/jonahwilliams/Documents/flutter/packages/flutter_tool_macos';
```

Comment:
It might help removing this hardcoding by bootstrapping with "builtin" extensions that live in the flutter_tool package.